bootstrap.sh

```diff
@@ -389,6 +389,11 @@ get_shell_startup_script() {
 install_sentry_cli() {
   log "Installing sentry-cli"
   if ! command -v sentry-cli &>/dev/null; then
+    # This ensures that sentry-cli has a directory to install under
+    [ ! -d /usr/local/bin ] && (
+      sudo_askpass mkdir /usr/local/bin;
+      sudo_askpass chown ${USER}:admin /usr/local/bin
```

Comment:
This is necessary for tools like Docker being able to install binaries inside of it.